stoplight-admin.gemspec

```diff
@@ -23,17 +23,17 @@ Gem::Specification.new do |gem|
     Dir.glob(File.join('lib', '**', '*.rb')) +
     Dir.glob(File.join('lib', '**', '*.haml'))
 
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 2.5.0'
 
   {
-    'haml' => '5.0.4',
-    'sinatra-contrib' => '2.2.3'
+    'haml' => '5.0',
+    'sinatra-contrib' => '3.0.0'
   }.each do |name, version|
     gem.add_dependency(name, "~> #{version}")
   end
 
   gem.add_dependency 'redis', '>= 3.2'
-  gem.add_dependency 'stoplight', '>= 1.4'
+  gem.add_dependency 'stoplight', '~> 3.0.0'
 
   {
     'bundler' => '1.10',
```

Comment:
@Lokideos would you mind bumping this dependency? Bundler 1.10 is almost 7 years old and it's incompatible with bundler 2.x

Reply:
@bolshakov since `bundler`, `pry-byebug`, `rake`, and `rspec` are development dependencies, I bumped all their versions to the latest versions.